### PR TITLE
Create menus for organization pages

### PR DESCRIPTION
**Pull request by lunny**
Gitea Theme Screenshots:

1 Organization Home

image

2 Members

image

3 Teams

image

4 Settings

image

Arc-green Theme Screenshots:

1 Organization Home

image

2 Members

image

3 Teams

image

4 Settings

image

### CODE REVIEW

File: templates/org/menu.tmpl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        {{if .IsOrganizationOwner}}
+            <div class="right menu">
+                <a class="{{if .PageIsSettingsOptions}}active{{end}} item" href="{{.OrgLink}}/settings">
+                {{svg "octicon-tools"}} {{.i18n.Tr "repo.settings"}}
+                </a>
+            </div>
+        {{end}}
+    </div>
+</div>
```
Comment (Gusted) [inline templates/org/menu.tmpl]: New line

**Comment by KN4CK3R** [comment]
lgtm
In a follow up PR we may allow users to see member/teams of an org like Github does.

**Comment by wxiaoguang** [comment]
The org avatar for org home is larger than other pages.

Can `org/home` also use the `org/header`?

**Comment by lunny** [comment]
> Details

Take a look at github's org home and other page. :)

**Review by wxiaoguang — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Comment by fnetX** [comment]
I just wanted to comment late here, that the redundancy of the peoples / teams parts (top menu and side menu, both include a link) looks a bit off at first glance. The details of the side-menu are handy, and the top menu a nice navigation feature, but still ... it feels a little weird to me.

**Comment by lunny** [comment]
> I just wanted to comment late here, that the redundancy of the peoples / teams parts (top menu and side menu, both include a link) looks a bit off at first glance. The details of the side-menu are handy, and the top menu a nice navigation feature, but still ... it feels a little weird to me.

Please file another issue, we can send other PR to improve it.

**Pull request merged**